src/tools/perf/lib/ucp_tests.cc

```diff
@@ -310,12 +310,16 @@ class ucp_perf_test_runner {
 
         ucx_perf_test_start_clock(&m_perf);
 
+#if _OPENMP
+#pragma omp barrier
+#endif
```

Comment:
pls add helper inline function

Reply:
you mean for every omp operation everywhere in ucx_perftest? or just barrier?

Reply:
just of the 3 lines of the barrier